modules/ossm-remove-cleanup.adoc

```diff
@@ -42,7 +42,7 @@ $ oc delete svc maistra-admission-controller -n openshift-operators
 +
 [source,terminal]
 ----
-$ oc delete -n openshift-operators daemonset/istio-node
+$ oc delete -n openshift-operators daemonset/istio-cni-node
```

Comment:
This doesn't match the JIRA.  Since this change is implemented in 2.2 and we still support earlier versions, I think we need to provide both commands.

For OSSM 2.1.x and earlier
 
$ oc delete -n openshift-operators daemonset/istio-node

For OSSM 2.2.0 and later
 
$ oc delete -n openshift-operators daemonset/istio-cni-node

Reply:
This revision has been applied.